website/docs/api/plugins/plugin-content-blog.mdx

```diff
@@ -130,6 +130,13 @@ type CreateFeedItemsFn = (params: {
 }) => Promise<BlogFeedItem[]>;
 ```
 
+#### `SortBlogPostsFn` {#SortBlogPostsFn}
+
+```ts
+// TODO document this type
```

Comment:
```suggestion
```
 
We don't document the `BlogPost` type yet, cf `CreateFeedItemsFn` above. 

We probably should but I'd consider it's not the responsibility of this PR to add this doc, because we need to think a bit about the public API surface we want to expose/document here.

We'll probably document this later but maybe not all the fields, because we don't necessarily want to lose the ability to rename/remove attributes (cf the `formattedDate` that we'd like to remove).

For now this is kind of undocumented on purpose, so that we consider it's not public API and we can still refactor in v3.x without considering it as a breaking change.

Reply:
remove  TODO for now